auth_server/authn/plugin_authn.go

```diff
@@ -0,0 +1,80 @@
+/*
+   Copyright 2019 Cesanta Software Ltd.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       https://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package authn
+
+import (
+	"fmt"
+	"github.com/cesanta/glog"
+	"os"
+	"plugin"
+)
+
+type PluginAuthnConfig struct {
+	PluginPath string `yaml:"plugin_path"`
+	Authn      Authenticator
```

Comment:
this should be an attribute of the PluginAuthn struct, not the config.

Reply:
Fixed